site/realm/data_sources/mongodb-atlas/config.json

```diff
@@ -3,6 +3,7 @@
     "type": "mongodb-atlas",
     "config": {
         "clusterName": "AIIDDev",
+        "namespacePreimageConfigs": [],
```

Comment:
![image](https://user-images.githubusercontent.com/1172479/194102742-d685eca1-a89e-410c-8a69-f5f226e4d10f.png)

What's this for?